packages/components/src/ListItem/ListItem.js

```diff
@@ -0,0 +1,38 @@
+import { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import { createStyleSheet } from '@platzily-ui/styling';
+
+const useStyleSheet = createStyleSheet((theme) => {
+  return {
+    root: {
+      display: 'flex',
+      alignItems: 'center',
+      padding: theme.spacing(2),
+      color: theme.palette.primary.main,
+      lineHeight: '1rem',
+      '&:hover': {
+        textDecoration: 'underline',
+      },
+      '&:focus': {
+        fontWeight: 'bold',
```

Comment:
Is this defined by the design system?

Reply:
Yes

Reply:
Ok I see. This refers to this.
![Screenshot from 2021-11-27 10-45-45](https://user-images.githubusercontent.com/30012444/143687928-f2264f07-d355-442d-b936-98a7ff246acc.png)

Reply:
But, there is a thing. This styles dont apply to an active state. But to the active module. For example if the user is in the Dashboard module this link item would show like this. If we only use the active, then when the user clicks away it would dissapear. Maybe a selected prop that applies the style? :thinking:

Reply:
![image](https://user-images.githubusercontent.com/51223315/143688549-fb79ad0f-2426-4097-be33-15d7d665a1a4.png)